examples/flutter_gallery/lib/demo/customized/snapping_scroll_physics.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+class SnappingScrollPhysics extends ClampingScrollPhysics {
+  const SnappingScrollPhysics({
+    ScrollPhysics parent,
+    @required this.midScrollOffset,
+  })  : assert(midScrollOffset != null),
+        super(parent: parent);
+
+  final double midScrollOffset;
+
+  @override
+  SnappingScrollPhysics applyTo(ScrollPhysics ancestor) {
+    return SnappingScrollPhysics(
+        parent: buildParent(ancestor), midScrollOffset: midScrollOffset);
+  }
+
+  @override
+  Simulation createBallisticSimulation(
```

Comment:
Generally fit params in one line or one param per line if it doesn't fit

Reply:
dartfmt fail